btr-web/btr-common-components/components/bcros/inputs/Address/Index.vue

```diff
@@ -104,9 +104,8 @@
 <script setup lang="ts">
 import { z } from 'zod'
 import { FormError } from '#ui/types'
-import { getAddressLine1Validator, getAddressCityValidator, getAddressRegionValidator, getAddressPostalCodeValidator }
-  from '../../../../../btr-main-app/utils/validators'
-
+import { getAddressCityValidator, getAddressLine1Validator, getAddressPostalCodeValidator, getAddressRegionValidator }
+  from '../../../../utils/validation/validators'
```

Comment:
I think you can omit the import here by adding an index file in the validation folder. At the very least you can reference it with '~/utils/validation/validators'

Reply:
There is an index file under the utils folder. I will just export these functions there.